receiver/oracledbreceiver/scraper.go

```diff
@@ -115,13 +170,21 @@ func newScraper(metricsBuilder *metadata.MetricsBuilder, metricsBuilderConfig me
 	return scraper.NewMetrics(s.scrape, scraper.WithShutdown(s.shutdown), scraper.WithStart(s.start))
 }
 
-func newLogsScraper(scrapeCfg scraperhelper.ControllerConfig, logger *zap.Logger, providerFunc dbProviderFunc, clientProviderFunc clientProviderFunc, instanceName string) (scraper.Logs, error) {
+func newLogsScraper(metricsBuilder *metadata.MetricsBuilder, metricsBuilderConfig metadata.MetricsBuilderConfig, scrapeCfg scraperhelper.ControllerConfig,
+	logger *zap.Logger, providerFunc dbProviderFunc, clientProviderFunc clientProviderFunc, instanceName string, metricCache *lru.Cache[string, map[string]int64],
+	topQueryCollectCfg TopQueryCollection, querySampleCfg QuerySample, hostName string) (scraper.Logs, error) {
 	s := &oracleScraper{
-		scrapeCfg:          scrapeCfg,
-		logger:             logger,
-		dbProviderFunc:     providerFunc,
-		clientProviderFunc: clientProviderFunc,
-		instanceName:       instanceName,
+		mb:                   metricsBuilder,
```

Comment:
i don't see metricsBuilder used for log scraper, and the name is a little bit confusing. maybe remove it from the function paramater?